src/fitnesse/reporting/SuiteHtmlFormatter.java

```diff
@@ -177,6 +215,79 @@ public void testSystemStarted(TestSystem testSystem) {
       writeData(insertScript.html());
     }
   }
+  
+
+  
+  protected void showAssertionResult(Assertion assertion, String testResult, boolean overwriteLastMessage) {
+    if(testSystemListenerShowInstructions >0){
+      try {
+        String instructionText;
+        String position;
+        Expectation expectation = assertion.getExpectation();
+        if (expectation instanceof TableCell) {
+          TableCell cell = (TableCell) expectation;
+          position = String.format("% 3d,% 3d", cell.getRow(), cell.getCol());
+        } else {
+          position = "       ";
+        }
+        assertion.getInstruction();
+        instructionText = assertion.getInstruction().toString();
+        instructionText = position + ": " + instructionText + "-->" + testResult;
+        instructionText = HtmlUtil.escapeHTML(instructionText);
+        if (overwriteLastMessage){
+          this.instructionsHtmlText.removeLast();
+        } else {
+          this.instructionsHtmlText.removeFirst();
+        }        
+        this.instructionsHtmlText.addLast(instructionText);
+        String allInstructionsText = this.instructionsHtmlText.stream().collect(Collectors.joining("<br>"));
+        String html = "<h4>"+allInstructionsText+"</h4>";
+        String insertScript = JavascriptUtil.makeReplaceElementScript("step-by-step-Id2", html).html();
+        writeData(insertScript);
+      } catch (Exception e){
+        String html = e.getMessage();
```

Comment:
This exception is ignored. I assume it is never expected to happen. Shouldn't it be logged  (so it shows up in the console where the wiki was started)?